blackbox/requirements.txt

```diff
@@ -5,7 +5,7 @@ Babel==2.4.0
 colorama==0.3.9
 crash==0.22.1
 crate==0.20.1
-crate-docs-theme==0.5.24
+crate-docs-theme
```

Comment:
We're always using the latest version here and the build artifacts get cleared regularly. So should be fine. Maybe add a small comment to clarify that?